api/mesh/v1alpha1/externalservice.proto

```diff
@@ -57,6 +57,8 @@ message ExternalService {
     }
 
     TLS tls = 2;
+
+    bool skipVIPGeneration = 3 [ (doc.required) = true ];
```

Comment:
I'm not sure it's actually required right?

Reply:
Please explain what this is for.

Reply:
how about `disableAutoDNS` ? At least I'd use `disable` instead of `skip` .

Maybe `AddressVipDisabled` this is to sound similar to the cp config: `ServiceVipEnabled`.

Reply:
I prefer `disable` over `skip`. And this is only for `host` right? So you still get the` external-service-name.mesh` entry. Maybe it should be called `disableHostDNS` or `disableAutoHostDNS`

Reply:
`disableAutoHostDNS` - it's a bit verbose but I think it describes the switch most accurately

Reply:
I like disable. Maybe `disableMeshHostDNS`?

Reply:
I like "disableHostDNS" and I would in fact make it something like "skipHostDNSEntry" (or I guess "disable") because "disable ... DNS" makes me think about like disabling DNS as a whole. To me it's not clear enough that it's for _other services_ that a DNS name isn't being generated for this ExternalService. Assuming I've understood correctly

Reply:
`Mesh` might mislead people that it disables `.mesh` - right?

Reply:
Yeah, I agree with the `entry` part. So `disableHostDNSEntry`?